src/App.tsx

```diff
@@ -20,12 +26,13 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                onChange={handleChange}
               />
             </div>
           </div>
         </div>
 
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList movies={moviesFromServer} query={query} />
```

Comment:
you should only pass here as props movies as filtered moviesFromServer, query is redundant so please remove it